augurpy/evaluate.py

```diff
@@ -213,7 +205,8 @@ def run_cross_validation(
     """
     scorer = set_scorer(estimator)
     x = subsample.to_df()
-    y = subsample.obs[[col for col in subsample.obs if col.startswith("y")]]
+    # taking the first of the two columns
```

Comment:
Think we should just ensure that we document this behavior.

Reply:
Sure! The values are minimally different. Before I dropped the first column when calculating the dummy variables automatically, so if I was taking the second column here the behaviour is exactly the same. I just thought people might wonder why Im taking the second and not the first column, which is why I changed it. 
It changes which labels are associated to positive and which to negative, I believe, and there are differences in cross validation because the splits will be different.